exercises/practice/pig-latin/test_pig_latin.c

```diff
@@ -2,210 +2,192 @@
 #include "pig_latin.h"
 #include <stdlib.h>
 
+static char *res = NULL;
 void setUp(void)
 {
 }
 
 void tearDown(void)
 {
+   if (res)
+      free(res);
+   res = NULL;
 }
 
 static void test_word_beginning_with_a(void)
 {
    const char *phrase = "apple";
-   char *res = translate(phrase);
+   res = translate(phrase);
    TEST_ASSERT_EQUAL_STRING("appleay", res);
```

Comment:
Potentially these lines could be further refactored out of each `test_` function to a  single `check_translation()` function.
Do you think this PR could cover that?

Reply:
Yes, I was thinking about that as I saw that done in another one of the leaking exercises. I could use that one as a template for this one and refactor. I may not get to it until tomorrow, but thanks for the go-ahead!

Reply:
pig latin now revised from roman numeral template.